snippets/go.json

```diff
@@ -156,6 +156,10 @@
 			"prefix": "tf",
 			"body": "func Test$1(t *testing.T) {\n\t$0\n}"
 		},
+		"benchmark function": {
+			"prefix": "bf",
+			"body": "func Benchmark$1(b *testing.B) {\n\tfor n := 0; n < b.N; n++ {\n\t\t$0\n\t}\n}"
```

Comment:
Allow to change the variable `n` in the for loop here just like the for loop snippet. And name it `i` or `index` to begin with.